shared/secrets_masker/src/airflow_shared/secrets_masker/secrets_masker.py

```diff
@@ -206,6 +202,8 @@ class SecretsMasker(logging.Filter):
     ALREADY_FILTERED_FLAG = "__SecretsMasker_filtered"
     MAX_RECURSION_DEPTH = 5
     _has_warned_short_secret = False
+    # Private variable with method access
+    _mask_secrets_in_logs = False
```

Comment:
Why is this a private variable, given we change its settings (False/True)

Reply:
Yeah it should instead be a public variable but user experience wise calling a method for setting / getting would be more convenient than: `SecretsMasker.mask_secrets_in_logs = True`

Ref: https://github.com/apache/airflow/pull/54449#discussion_r2291331166